docs/installing.rst

```diff
@@ -148,8 +148,22 @@ The following virtualization providers are suggested for the respective operatin
     # Set libvirtd to start automatically on system boot
     $ sudo systemctl enable libvirtd
 
+---------------------------------------------------
+3. Install dependencies for vagrant-service-manager
```

Comment:
Should we document these dependencies in the VSM repo and reference them from here?  I think we have to document these dependencies in VSM regardless.